src/modules/MathematicalExpresionsModule.js

```diff
@@ -24,11 +23,10 @@ class MathematicalExpresionsModule extends Module {
         try {
             const question = math.evaluate(message.content.toLowerCase());
 
-            if (question != undefined && question != message.content && !(question instanceof Function)) 
+            if (question != undefined && question != message.content && !(question instanceof Function) && !(message.content.includes("\"")))
```

Comment:
Brackets are not required in this case. Remove them. 
```suggestion
            if (question != undefined && question != message.content && !(question instanceof Function) && !message.content.includes("\""))
```